Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -105,8 +116,16 @@ def run
 
                 - run: brew test-bot --only-tap-syntax
 
-                - run: brew test-bot --only-formulae#{" --root-url='#{root_url}'" if root_url}
+                - name: Run tests
                   if: github.event_name == 'pull_request'
+                  env:
+                    TOKEN: ${{ github.token }}
+                  run: |
+                    base64_token=$(echo -n "${TOKEN}" | base64 | tr -d "\\n")
+                    echo "::add-mask::${base64_token}"
+                    export "HOMEBREW_DOCKER_REGISTRY_TOKEN=${base64_token}"
```

Comment:
Might be nice to split this into another step?

Reply:
Makes sense!